clouddriver-openstack/src/main/groovy/com/netflix/spinnaker/clouddriver/openstack/provider/ImageProvider.groovy

```diff
@@ -0,0 +1,23 @@
+/*
+ * Copyright 2016 Target, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.netflix.spinnaker.clouddriver.openstack.provider
+
+import com.netflix.spinnaker.clouddriver.openstack.model.OpenstackImage
+
+interface ImageProvider {
+  Map<String, Set<OpenstackImage>> listImagesByAccount()
```

Comment:
Do you want it to return `Image`s instead of `OpenstackImage`s? From a practical standpoint, it doesn't make much difference, but there is no reason to tie this to the implimenation